generated/nifake/nifake_service.cpp

```diff
@@ -86,7 +86,22 @@ namespace nifake_grpc {
       return ::grpc::Status::CANCELLED;
     }
     try {
-      return ::grpc::Status(::grpc::UNIMPLEMENTED, "TODO: This server handler has not been implemented.");
+      auto vi_grpc_session = request->vi();
```

Comment:
Can you add an auto-test for this method now that it's supported so we don't regress on boolean array outputs? If you look at `NiFakeService_ReturnListOfDurationsInSeconds_CallsReturnListOfDurationsInSeconds` in `source/tests/unit/ni_fake_service_tests.cpp` it should be very similar except it will have a `ViBoolean` and `bool` array instead of `ViReal64` and `double`.

Reply:
Good catch @reckenro!

Reply:
Added unit test.